### DISCUSSION TITLE
Lab3 Opt1Baseline: there is no xrt.run_summary after SW emulation.

### DISCUSSION THREAD

**Discussion by poppin-mice** [bol-edu/HLS-SOC-Discussions#18] [Q&A] [answered]
I have followed the workbook and finish until page 16 using the u50_03. I found that only xclbin.run_summary is generated. However, in this xclbin.run_summary, I can only read the timeline report. There is no profile summary or run guidance in the report as shown in the workbook. The following is the screenshot of the xclbin.run_summary:
![image](https://user-images.githubusercontent.com/11053589/191908105-12021648-310d-4c75-8ab7-3ad118f91726.png)

This is the overview of the vitis project:
![image](https://user-images.githubusercontent.com/11053589/191908198-512b2560-3486-44ec-b454-a43f1a19c33b.png)

And this is the setting of the run configuration:
![image](https://user-images.githubusercontent.com/11053589/191908327-ca0abbd8-bd62-45fb-a46f-816b9ad7e1ac.png)
![image](https://user-images.githubusercontent.com/11053589/191908354-4ba36e49-c86c-4f4c-8a28-24c814e0338f.png)
![image](https://user-images.githubusercontent.com/11053589/191908484-5eab2cff-b67b-4171-abcd-4084fd0ea363.png)

I think this may be caused by the incorrect settings of the overall project or the board.

**Comment by poppin-mice** [comment]
I re-run the lab on u50_01, and it output the correct files (xrt.run_summary). I am wondering why this happens.

**Comment by ciwade870307** [comment, marked as answer]
Thank you for your feedback. We have updated the software. u50_03 and u50_4 can output the profile summary now.

**Reply by poppin-mice** [reply]
Thanks for the help!
